librz/core/canalysis.c

```diff
@@ -378,9 +378,20 @@ static void bb_info_print(RzCore *core, RzAnalysisFunction *fcn, RzAnalysisBlock
 		inputs += (bb2->jump == bb->addr) + (bb2->fail == bb->addr);
 	}
 	if (bb->switch_op) {
-		RzList *unique_cases = rz_list_uniq(bb->switch_op->cases, casecmp);
-		outputs += rz_list_length(unique_cases);
-		rz_list_free(unique_cases);
+		if (bb->switch_op) {
```

Comment:
looks like you also have dup this check.

Reply:
Argh, that's a mistake by my side. Apologies.